index.md

```diff
@@ -58,6 +58,9 @@ Representatives of Shared Savings Program ACOs are now invited to express their
 
   * **Why are Shared Savings Program ACOs getting access to this data?**
     * As indicated in the regulation [§425.704](https://www.ecfr.gov/cgi-bin/text-idx?SID=20d60582b65758d4ace3a6b5291cfc4f&mc=true&node=se42.3.425_1704&rgn=div8){:target="_blank"}, “subject to providing the beneficiary with the opportunity to decline data sharing as described in this §425.708, and subject to having a valid DUA in place, CMS, upon the ACO's request for the data for purposes of evaluating the performance of its ACO participants or its ACO providers/suppliers, conducting quality assessment and improvement activities, and conducting population-based activities relating to improved health, will provide the ACO with beneficiary identifiable claims data for preliminarily prospectively and prospectively assigned beneficiaries and other beneficiaries who receive primary care services from an ACO participant that submits claims for primary care services used to determine the ACO's assigned population under subpart E of this part during the performance year.”
+    
+  * **What claims does BCDA exclude?**
```

Comment:
Since the question is "What claims does BCDA exclude?", do we also need to mention something about exclusion of claims for patients who have opted out of sharing via 1-800-Medicare?

Reply:
Great question - let me check with Anna.

Reply:
Just found approved language in our talking points! Will add in.